pyproject.toml

```diff
@@ -1,3 +1,7 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
```

Comment:
If `build-system` is missing, the following config is used:

https://pip.pypa.io/en/stable/reference/build-system/pyproject-toml/#fallback-behaviour

```
[build-system]
requires = ["setuptools>=40.8.0"]
build-backend = "setuptools.build_meta:__legacy__"
```